core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -176,7 +176,14 @@
   /**
    * The default timeout in nanos for a keepalive ping request.
    */
-  public static final long DEFAULT_KEEPALIVE_TIMEOUT_NANOS = TimeUnit.MINUTES.toNanos(2);
+  public static final long DEFAULT_KEEPALIVE_TIMEOUT_NANOS = TimeUnit.SECONDS.toNanos(20L);
```

Comment:
The default should be 20 seconds. The old value 2 minutes was used for OkHttp, should we keep that value for OkHttp @ejona86 @zsurocking ?